assets/src/application/ui/layout/modal/ModalWithAlert.tsx

```diff
@@ -19,8 +19,10 @@ const ModalWithAlert: React.FC<Props> = ({ children, showAlertOnEscape, ...props
 	const { isOpen, onOpen, onClose } = useDisclosure();
 	const cancelRef = React.useRef();
 	const cancelBtnText = props.cancelBtnText || __('No');
+	const cancelButton = <Button buttonText={cancelBtnText} ref={cancelRef} onClick={onClose} />;
 	const header = props.header || __('Are you sure you want to close this?');
 	const okBtnText = props.okBtnText || __('Yes');
+	const okButton = <Button buttonText={okBtnText} buttonType={ButtonType.ACCENT} onClick={props.onClose} ml={3} />;
```

Comment:
what about using the `useConfirmationDialog` hook that you just added ?

Reply:
will do

Reply:
double tested after change request:


![Peek 2020-05-06 22-52](https://user-images.githubusercontent.com/1477453/81222069-6521f700-8fec-11ea-9674-788a88a750a6.gif)

Reply:
the update: https://github.com/eventespresso/event-espresso-core/pull/2777/commits/3f4a54f4406e86aaca97a47ac44423ab0f8c2268